explorer/src/lib/chain-info/addCountAndUnique.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Returns an array of unique objects, sorted by the frequency of their occurrence, with an additional count property representing the number of times that city-country combination appeared in the input data.
+ *
+ * @param {{city: string, country: string, countryCode: string, lat: number, lon: number}[]} data
```

Comment:
This could use the `NodeLocation` type that you have already defined.

Reply:
It cannot as the `count` property is missing.

Reply:
You can make `count` optional in the type declaration. Can check in a bit.

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys